test/legacy/legacy-default-xpring-client-test.ts

```diff
@@ -35,6 +35,23 @@ const transactionStatusFailureCodes = [
 
 const transactionHash = 'DEADBEEF'
 
+/**
```

Comment:
Definitely don't write this class twice. 

You could just make it's own file in `tests/fakes/fake-grpc-error.ts`) or stick it in `fake-network-client`. I'd probably recommend the former as `fake-network-client` is already becoming pretty long, but don't have strong opinions if you want to keep all the fake network stuff as one logical unit.

Reply:
great, this is in its own file now.